### PR TITLE
Add Kleptomania

### PR DESCRIPTION
**Pull request by HariMahi**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -431,6 +432,20 @@ context. There have been no treatment methods discovered as of now.
 
 [R: National Institute of Neurological Disorders and Stroke](https://www.ninds.nih.gov/Disorders/All-Disorders/Prosopagnosia-Information-Page) 
 
+#### [Kleptomania](https://en.wikipedia.org/wiki/Kleptomania)
+
+> An impulse control disorder that results in an irresistible urge to steal
+
+Kleptomania or klopemania is the inability to refrain from the urge
```
Comment (icyflame) [inline readme.md:439]: Fix grammar. Paragraph should be longer

File: readme.md [hunk 2]
```diff
@@ -431,6 +432,20 @@ context. There have been no treatment methods discovered as of now.
...
+> An impulse control disorder that results in an irresistible urge to steal
+
+Kleptomania or klopemania is the inability to refrain from the urge 
+for stealing items and is usually done for reasons other than personal
+use or financial gain. First described in 1816, kleptomania is classified
+in psychiatry as an impulse control disorder.
+
+[R: PsychologyToday](https://www.psychologytoday.com/conditions/kleptomania)
+
+[R: minddisorders.com](http://www.minddisorders.com/Kau-Nu/Kleptomania.html)
```
Comment (icyflame) [inline readme.md:446]: Please add one more ref. minddisorders.com is rather short.

**Review by icyflame — CHANGES_REQUESTED** [review]

**Comment by HariMahi** [comment]
Review the PRs

On Mon, Dec 25, 2017 at 10:36 PM, Siddharth Kannan <notifications@github.com

**Comment by icyflame** [comment]
This needs more work. The current paragraph is not satisfactory.

**Pull request closed without merging**